App.tsx

```diff
@@ -7,13 +7,15 @@ import { AsyncStorage } from "react-native";
 
 import Home from "./src/components/pages/Home";
 import LogIn from "./src/components/pages/LogIn";
+import OnboardingCamera from "./src/components/pages/OnboardingCamera";
 import Onboarding from "./src/components/pages/Onboarding";
-import createStore, { store, persistor } from "./src/store";
```

Comment:
Sorry for that, I'll add some CI testing in a bit. I disabled the lint checks for unused imports before, but it would make sense if we turned that on again I think.